src/frontend/tsx/Chessboard.tsx

```diff
@@ -375,12 +374,7 @@ export function Chessboard () {
 
   return (
     <div className='game-container'>
-      <div className='title-container'>
-        <p className='title-text' style={{ fontSize: URBIT_CHESS.lengthToFontSize.get(boardTitle.length) }}>
-          {`${boardTitle}`}
-        </p>
-      </div>
-      <div className={`board-container ${boardTheme} ${pieceTheme}`}>
+      <div className={`board-container ${boardTheme} ${pieceTheme}`} style={{ pointerEvents: (reviewMode ? 'none' : 'auto') }}>
```

Comment:
`reviewMode` and messing around with the current position isn't what we should do. What we should be doing is:
1. Keep the current position, as far as the `chess` object (which implements all of the frontend chess logic) is concerned
2. Display the old position using Chessground
3. Make board `viewOnly` in the [Chessground configs](https://github.com/lichess-org/chessground/blob/master/src/config.ts) when browsing an old position

To accomplish 1, we just need to not touch the board position (which is why I deleted `reviewPosition` in the above comment).

To accomplish 2, the value for the `fen` Chessground config on line `185` should be: `fen: activeGameMoves.get(displayGame.gameId).get(displayIndex)`. (Note that this may require some fiddling, since the actual point of that line is to update the Chessground board to display the move that we just made inside of the `chess` object which actually records our chess state).

To accomplish 3, we would need to compute a new "non-state constant" (see line `50`) for whether the board is `viewOnly` (whether `displayIndex` is the final index of the `Array` of moves for this game or not). Then, we would set the `viewOnly` config in `updateBoard` just like we set the other configs (e.g. `fen`, `turnColor`, etc.).

Additional notes:
- Using the above implementation, we need to be mindful to increment `displayIndex` if it's at the latest move and we receive/make a new move.
- If we just started a game, the `Array` of `Move`s might be empty, so we need to handle the base case of a `displayIndex` of `0`.